.github/workflows/test.yml

```
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
```

Could we open this up so that folks can ensure their potential PR builds prior to submission?
```suggestion
    branches: '*'
```

Many projects do the same for `pull_request.branches` below, which would be my preference, but, I believe, that can lead to double builds on pushed updates to existing PRs.  Since these builds are only 2-3 minutes, I think the additional builds aren't a big deal.